LoopKitUI/Views/QuantityScheduleEditor.swift

```diff
@@ -119,9 +119,9 @@ struct QuantityScheduleEditor<ActionAreaContent: View>: View {
         HStack { // to align with guardrail warning, if present
             VStack (alignment: .leading, spacing: 20) {
                 Text(LocalizedString("You can edit a setting by tapping into any line item.", comment: "Description of how to edit setting"))
-                Text(LocalizedString("You can add entries for different times of day by using the [+].", comment: "Description of how to add a range"))
+                Text(LocalizedString("You can add entries for different times of day by using the ➕.", comment: "Description of how to add a range"))
             }
-            .foregroundColor(.accentColor)
+            .foregroundColor(.secondary)
             .font(.subheadline)
```

Comment:
[nit] If this formatting is common and consistent across these messages, you might consider extracting it to a shared ViewModifier

It would simplify cases like this PR where each occurrence needs to be updated to match some design change